### DISCUSSION TITLE
Publish Error

### DISCUSSION THREAD

**Discussion by jefflim69** [naverz/zepeto-studio-kor#1025] [5. Q&A]
World test 를 하기 위해 Publish 하여 올리려 하는 데 , 아래와 같이 에러 발생하며 올릴 수가 없네요 .
![Error_msg](https://user-images.githubusercontent.com/12184282/179384602-c706e1da-29d7-4baf-80d9-1162f53b86d4.PNG)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, https://github.com/naverz/zepeto-studio-kor/discussions/476#discussioncomment-2328167 이슈와 같은 원인으로 추정됩니다.
유니티를 종료하시고 프로젝트의 Library 폴더, obj폴더를 삭제하신 후 다시 실행해서 빌드 해보세요.
